shared/plan/service.py

```diff
@@ -311,3 +313,42 @@ def has_seats_left(self) -> bool:
             self.plan_activated_users is None
             or len(self.plan_activated_users) < self.plan_user_count
         )
+
+    @property
+    def is_enterprise_plan(self) -> bool:
```

Comment:
nit: can just do `return self.plan_name in ...` for all these

Reply:
good point, will update